2024-10-29 17:05:49.832 [main] ERROR o.s.b.diagnostics.LoggingFailureAnalysisReporter - [] 

***************************
APPLICATION FAILED TO START
***************************

Description:

Failed to configure a DataSource: 'url' attribute is not specified and no embedded datasource could be configured.

Reason: Failed to determine a suitable driver class


Action:

Consider the following:
	If you want an embedded database (H2, HSQL or Derby), please put it on the classpath.
	If you have database settings to be loaded from a particular profile you may need to activate it (no profiles are currently active).

2024-10-29 17:06:23.462 [main] ERROR o.s.b.diagnostics.LoggingFailureAnalysisReporter - [] 

***************************
APPLICATION FAILED TO START
***************************

Description:

Failed to configure a DataSource: 'url' attribute is not specified and no embedded datasource could be configured.

Reason: Failed to determine a suitable driver class


Action:

Consider the following:
	If you want an embedded database (H2, HSQL or Derby), please put it on the classpath.
	If you have database settings to be loaded from a particular profile you may need to activate it (no profiles are currently active).

2024-10-29 17:09:39.275 [tio-group-5] ERROR org.tio.core.ReadCompletionHandler - [] server:0.0.0.0:9326, client:172.16.85.134:63609, javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknownjava.nio.HeapByteBuffer[pos=30 lim=30 cap=30]
javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknown
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:131)
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:117)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:365)
	at java.base/sun.security.ssl.Alert$AlertConsumer.consume(Alert.java:293)
	at java.base/sun.security.ssl.TransportContext.dispatch(TransportContext.java:204)
	at java.base/sun.security.ssl.SSLTransport.decode(SSLTransport.java:172)
	at java.base/sun.security.ssl.SSLEngineImpl.decode(SSLEngineImpl.java:736)
	at java.base/sun.security.ssl.SSLEngineImpl.readRecord(SSLEngineImpl.java:691)
	at java.base/sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:506)
	at java.base/sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:482)
	at java.base/javax.net.ssl.SSLEngine.unwrap(SSLEngine.java:679)
	at org.tio.core.ssl.facade.Worker.doUnwrap(Worker.java:276)
	at org.tio.core.ssl.facade.Worker.unwrap(Worker.java:389)
	at org.tio.core.ssl.facade.Worker.unwrap(Worker.java:425)
	at org.tio.core.ssl.facade.SSLFacade.decrypt(SSLFacade.java:272)
	at org.tio.core.ReadCompletionHandler.completed(ReadCompletionHandler.java:281)
	at org.tio.core.ReadCompletionHandler.completed(ReadCompletionHandler.java:212)
	at java.base/sun.nio.ch.Invoker.invokeUnchecked(Invoker.java:129)
	at java.base/sun.nio.ch.Invoker$2.run(Invoker.java:221)
	at java.base/sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:113)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-10-29 17:09:39.314 [https-jsse-nio-8089-exec-7] ERROR com.dot.msg.chat.tio.util.TioUtil - [5f3877765fac4788ac74d10576aaff31] 用户处于离线状态,userId:3
2024-10-29 17:09:39.315 [https-jsse-nio-8089-exec-7] ERROR com.dot.msg.chat.tio.util.TioUtil - [5f3877765fac4788ac74d10576aaff31] 用户处于离线状态,userId:3
2024-10-29 17:10:09.070 [tio-group-26] ERROR org.tio.core.ReadCompletionHandler - [] server:0.0.0.0:9326, client:172.16.85.134:63778, javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknownjava.nio.HeapByteBuffer[pos=30 lim=30 cap=30]
javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknown
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:131)
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:117)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:365)
	at java.base/sun.security.ssl.Alert$AlertConsumer.consume(Alert.java:293)
	at java.base/sun.security.ssl.TransportContext.dispatch(TransportContext.java:204)
	at java.base/sun.security.ssl.SSLTransport.decode(SSLTransport.java:172)
	at java.base/sun.security.ssl.SSLEngineImpl.decode(SSLEngineImpl.java:736)
	at java.base/sun.security.ssl.SSLEngineImpl.readRecord(SSLEngineImpl.java:691)
	at java.base/sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:506)
	at java.base/sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:482)
	at java.base/javax.net.ssl.SSLEngine.unwrap(SSLEngine.java:679)
	at org.tio.core.ssl.facade.Worker.doUnwrap(Worker.java:276)
	at org.tio.core.ssl.facade.Worker.unwrap(Worker.java:389)
	at org.tio.core.ssl.facade.Worker.unwrap(Worker.java:425)
	at org.tio.core.ssl.facade.SSLFacade.decrypt(SSLFacade.java:272)
	at org.tio.core.ReadCompletionHandler.completed(ReadCompletionHandler.java:281)
	at org.tio.core.ReadCompletionHandler.completed(ReadCompletionHandler.java:212)
	at java.base/sun.nio.ch.Invoker.invokeUnchecked(Invoker.java:129)
	at java.base/sun.nio.ch.Invoker$2.run(Invoker.java:221)
	at java.base/sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:113)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-10-29 17:10:48.767 [tio-group-71] ERROR org.tio.core.ReadCompletionHandler - [] server:0.0.0.0:9326, client:172.16.85.134:63994, javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknownjava.nio.HeapByteBuffer[pos=30 lim=30 cap=30]
javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknown
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:131)
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:117)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:365)
	at java.base/sun.security.ssl.Alert$AlertConsumer.consume(Alert.java:293)
	at java.base/sun.security.ssl.TransportContext.dispatch(TransportContext.java:204)
	at java.base/sun.security.ssl.SSLTransport.decode(SSLTransport.java:172)
	at java.base/sun.security.ssl.SSLEngineImpl.decode(SSLEngineImpl.java:736)
	at java.base/sun.security.ssl.SSLEngineImpl.readRecord(SSLEngineImpl.java:691)
	at java.base/sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:506)
	at java.base/sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:482)
	at java.base/javax.net.ssl.SSLEngine.unwrap(SSLEngine.java:679)
	at org.tio.core.ssl.facade.Worker.doUnwrap(Worker.java:276)
	at org.tio.core.ssl.facade.Worker.unwrap(Worker.java:389)
	at org.tio.core.ssl.facade.Worker.unwrap(Worker.java:425)
	at org.tio.core.ssl.facade.SSLFacade.decrypt(SSLFacade.java:272)
	at org.tio.core.ReadCompletionHandler.completed(ReadCompletionHandler.java:281)
	at org.tio.core.ReadCompletionHandler.completed(ReadCompletionHandler.java:212)
	at java.base/sun.nio.ch.Invoker.invokeUnchecked(Invoker.java:129)
	at java.base/sun.nio.ch.Invoker$2.run(Invoker.java:221)
	at java.base/sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:113)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-10-29 17:15:14.464 [tio-group-226] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 接收用户处于离线状态,toUserId:2
2024-10-29 17:15:14.464 [tio-group-226] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 用户处于离线状态,userId:2
2024-10-29 17:15:14.647 [tio-group-226] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 接收用户处于离线状态,toUserId:2
2024-10-29 17:15:17.024 [tio-group-230] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 接收用户处于离线状态,toUserId:2
2024-10-29 17:23:11.316 [tio-group-107] ERROR org.tio.core.ReadCompletionHandler - [] server:0.0.0.0:9326, client:172.16.85.134:51362, javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknownjava.nio.HeapByteBuffer[pos=30 lim=30 cap=30]
javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknown
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:131)
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:117)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:365)
	at java.base/sun.security.ssl.Alert$AlertConsumer.consume(Alert.java:293)
	at java.base/sun.security.ssl.TransportContext.dispatch(TransportContext.java:204)
	at java.base/sun.security.ssl.SSLTransport.decode(SSLTransport.java:172)
	at java.base/sun.security.ssl.SSLEngineImpl.decode(SSLEngineImpl.java:736)
	at java.base/sun.security.ssl.SSLEngineImpl.readRecord(SSLEngineImpl.java:691)
	at java.base/sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:506)
	at java.base/sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:482)
	at java.base/javax.net.ssl.SSLEngine.unwrap(SSLEngine.java:679)
	at org.tio.core.ssl.facade.Worker.doUnwrap(Worker.java:276)
	at org.tio.core.ssl.facade.Worker.unwrap(Worker.java:389)
	at org.tio.core.ssl.facade.Worker.unwrap(Worker.java:425)
	at org.tio.core.ssl.facade.SSLFacade.decrypt(SSLFacade.java:272)
	at org.tio.core.ReadCompletionHandler.completed(ReadCompletionHandler.java:281)
	at org.tio.core.ReadCompletionHandler.completed(ReadCompletionHandler.java:212)
	at java.base/sun.nio.ch.Invoker.invokeUnchecked(Invoker.java:129)
	at java.base/sun.nio.ch.Invoker$2.run(Invoker.java:221)
	at java.base/sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:113)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-10-29 17:23:31.388 [tio-group-122] ERROR org.tio.core.ReadCompletionHandler - [] server:0.0.0.0:9326, client:172.16.85.134:51475, javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknownjava.nio.HeapByteBuffer[pos=30 lim=30 cap=30]
javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknown
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:131)
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:117)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:365)
	at java.base/sun.security.ssl.Alert$AlertConsumer.consume(Alert.java:293)
	at java.base/sun.security.ssl.TransportContext.dispatch(TransportContext.java:204)
	at java.base/sun.security.ssl.SSLTransport.decode(SSLTransport.java:172)
	at java.base/sun.security.ssl.SSLEngineImpl.decode(SSLEngineImpl.java:736)
	at java.base/sun.security.ssl.SSLEngineImpl.readRecord(SSLEngineImpl.java:691)
	at java.base/sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:506)
	at java.base/sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:482)
	at java.base/javax.net.ssl.SSLEngine.unwrap(SSLEngine.java:679)
	at org.tio.core.ssl.facade.Worker.doUnwrap(Worker.java:276)
	at org.tio.core.ssl.facade.Worker.unwrap(Worker.java:389)
	at org.tio.core.ssl.facade.Worker.unwrap(Worker.java:425)
	at org.tio.core.ssl.facade.SSLFacade.decrypt(SSLFacade.java:272)
	at org.tio.core.ReadCompletionHandler.completed(ReadCompletionHandler.java:281)
	at org.tio.core.ReadCompletionHandler.completed(ReadCompletionHandler.java:212)
	at java.base/sun.nio.ch.Invoker.invokeUnchecked(Invoker.java:129)
	at java.base/sun.nio.ch.Invoker$2.run(Invoker.java:221)
	at java.base/sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:113)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-10-29 17:24:53.100 [tio-group-167] ERROR org.tio.core.ReadCompletionHandler - [] server:0.0.0.0:9326, client:172.16.85.134:51855, javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknownjava.nio.HeapByteBuffer[pos=30 lim=30 cap=30]
javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknown
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:131)
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:117)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:365)
	at java.base/sun.security.ssl.Alert$AlertConsumer.consume(Alert.java:293)
	at java.base/sun.security.ssl.TransportContext.dispatch(TransportContext.java:204)
	at java.base/sun.security.ssl.SSLTransport.decode(SSLTransport.java:172)
	at java.base/sun.security.ssl.SSLEngineImpl.decode(SSLEngineImpl.java:736)
	at java.base/sun.security.ssl.SSLEngineImpl.readRecord(SSLEngineImpl.java:691)
	at java.base/sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:506)
	at java.base/sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:482)
	at java.base/javax.net.ssl.SSLEngine.unwrap(SSLEngine.java:679)
	at org.tio.core.ssl.facade.Worker.doUnwrap(Worker.java:276)
	at org.tio.core.ssl.facade.Worker.unwrap(Worker.java:389)
	at org.tio.core.ssl.facade.Worker.unwrap(Worker.java:425)
	at org.tio.core.ssl.facade.SSLFacade.decrypt(SSLFacade.java:272)
	at org.tio.core.ReadCompletionHandler.completed(ReadCompletionHandler.java:281)
	at org.tio.core.ReadCompletionHandler.completed(ReadCompletionHandler.java:212)
	at java.base/sun.nio.ch.Invoker.invokeUnchecked(Invoker.java:129)
	at java.base/sun.nio.ch.Invoker$2.run(Invoker.java:221)
	at java.base/sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:113)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-10-29 17:25:22.608 [tio-group-189] ERROR org.tio.core.ReadCompletionHandler - [] server:0.0.0.0:9326, client:172.16.85.134:52017, javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknownjava.nio.HeapByteBuffer[pos=30 lim=30 cap=30]
javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknown
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:131)
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:117)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:365)
	at java.base/sun.security.ssl.Alert$AlertConsumer.consume(Alert.java:293)
	at java.base/sun.security.ssl.TransportContext.dispatch(TransportContext.java:204)
	at java.base/sun.security.ssl.SSLTransport.decode(SSLTransport.java:172)
	at java.base/sun.security.ssl.SSLEngineImpl.decode(SSLEngineImpl.java:736)
	at java.base/sun.security.ssl.SSLEngineImpl.readRecord(SSLEngineImpl.java:691)
	at java.base/sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:506)
	at java.base/sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:482)
	at java.base/javax.net.ssl.SSLEngine.unwrap(SSLEngine.java:679)
	at org.tio.core.ssl.facade.Worker.doUnwrap(Worker.java:276)
	at org.tio.core.ssl.facade.Worker.unwrap(Worker.java:389)
	at org.tio.core.ssl.facade.Worker.unwrap(Worker.java:425)
	at org.tio.core.ssl.facade.SSLFacade.decrypt(SSLFacade.java:272)
	at org.tio.core.ReadCompletionHandler.completed(ReadCompletionHandler.java:281)
	at org.tio.core.ReadCompletionHandler.completed(ReadCompletionHandler.java:212)
	at java.base/sun.nio.ch.Invoker.invokeUnchecked(Invoker.java:129)
	at java.base/sun.nio.ch.Invoker$2.run(Invoker.java:221)
	at java.base/sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:113)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-10-29 17:25:27.335 [tio-group-200] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 接收用户处于离线状态,toUserId:2
2024-10-29 17:25:27.341 [tio-group-200] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 用户处于离线状态,userId:2
2024-10-29 17:25:27.518 [tio-group-200] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 接收用户处于离线状态,toUserId:2
2024-10-29 17:25:29.305 [tio-group-206] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 接收用户处于离线状态,toUserId:2
2024-10-29 17:25:31.000 [tio-group-211] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 接收用户处于离线状态,toUserId:2
2024-10-29 17:25:31.000 [tio-group-211] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 用户处于离线状态,userId:2
2024-10-29 17:25:31.165 [tio-group-211] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 接收用户处于离线状态,toUserId:2
2024-10-29 17:25:32.183 [tio-group-216] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 接收用户处于离线状态,toUserId:2
2024-10-29 17:26:42.689 [tio-group-43] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 接收用户处于离线状态,toUserId:2
2024-10-29 17:26:42.691 [tio-group-43] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 用户处于离线状态,userId:2
2024-10-29 17:26:42.998 [tio-group-43] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 接收用户处于离线状态,toUserId:2
2024-10-29 17:26:44.579 [tio-group-48] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 接收用户处于离线状态,toUserId:2
2024-10-29 17:26:46.847 [tio-group-53] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 接收用户处于离线状态,toUserId:2
2024-10-29 17:26:46.847 [tio-group-53] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 用户处于离线状态,userId:2
2024-10-29 17:26:47.013 [tio-group-53] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 接收用户处于离线状态,toUserId:2
2024-10-29 17:26:48.136 [tio-group-58] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 接收用户处于离线状态,toUserId:2
2024-10-29 17:33:01.841 [tio-group-75] ERROR org.tio.core.ReadCompletionHandler - [] server:0.0.0.0:9326, client:172.16.85.134:54301, javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknownjava.nio.HeapByteBuffer[pos=30 lim=30 cap=30]
javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknown
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:131)
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:117)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:365)
	at java.base/sun.security.ssl.Alert$AlertConsumer.consume(Alert.java:293)
	at java.base/sun.security.ssl.TransportContext.dispatch(TransportContext.java:204)
	at java.base/sun.security.ssl.SSLTransport.decode(SSLTransport.java:172)
	at java.base/sun.security.ssl.SSLEngineImpl.decode(SSLEngineImpl.java:736)
	at java.base/sun.security.ssl.SSLEngineImpl.readRecord(SSLEngineImpl.java:691)
	at java.base/sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:506)
	at java.base/sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:482)
	at java.base/javax.net.ssl.SSLEngine.unwrap(SSLEngine.java:679)
	at org.tio.core.ssl.facade.Worker.doUnwrap(Worker.java:276)
	at org.tio.core.ssl.facade.Worker.unwrap(Worker.java:389)
	at org.tio.core.ssl.facade.Worker.unwrap(Worker.java:425)
	at org.tio.core.ssl.facade.SSLFacade.decrypt(SSLFacade.java:272)
	at org.tio.core.ReadCompletionHandler.completed(ReadCompletionHandler.java:281)
	at org.tio.core.ReadCompletionHandler.completed(ReadCompletionHandler.java:212)
	at java.base/sun.nio.ch.Invoker.invokeUnchecked(Invoker.java:129)
	at java.base/sun.nio.ch.Invoker$2.run(Invoker.java:221)
	at java.base/sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:113)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-10-29 17:33:24.794 [tio-group-96] ERROR org.tio.core.ReadCompletionHandler - [] server:0.0.0.0:9326, client:172.16.85.134:54444, javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknownjava.nio.HeapByteBuffer[pos=30 lim=30 cap=30]
javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknown
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:131)
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:117)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:365)
	at java.base/sun.security.ssl.Alert$AlertConsumer.consume(Alert.java:293)
	at java.base/sun.security.ssl.TransportContext.dispatch(TransportContext.java:204)
	at java.base/sun.security.ssl.SSLTransport.decode(SSLTransport.java:172)
	at java.base/sun.security.ssl.SSLEngineImpl.decode(SSLEngineImpl.java:736)
	at java.base/sun.security.ssl.SSLEngineImpl.readRecord(SSLEngineImpl.java:691)
	at java.base/sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:506)
	at java.base/sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:482)
	at java.base/javax.net.ssl.SSLEngine.unwrap(SSLEngine.java:679)
	at org.tio.core.ssl.facade.Worker.doUnwrap(Worker.java:276)
	at org.tio.core.ssl.facade.Worker.unwrap(Worker.java:389)
	at org.tio.core.ssl.facade.Worker.unwrap(Worker.java:425)
	at org.tio.core.ssl.facade.SSLFacade.decrypt(SSLFacade.java:272)
	at org.tio.core.ReadCompletionHandler.completed(ReadCompletionHandler.java:281)
	at org.tio.core.ReadCompletionHandler.completed(ReadCompletionHandler.java:212)
	at java.base/sun.nio.ch.Invoker.invokeUnchecked(Invoker.java:129)
	at java.base/sun.nio.ch.Invoker$2.run(Invoker.java:221)
	at java.base/sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:113)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-10-29 17:33:36.488 [tio-group-112] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 接收用户处于离线状态,toUserId:1
2024-10-29 17:33:36.489 [tio-group-112] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 用户处于离线状态,userId:1
2024-10-29 17:33:36.489 [tio-group-112] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 接收用户处于离线状态,toUserId:2
2024-10-29 17:33:36.489 [tio-group-112] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 用户处于离线状态,userId:2
2024-10-29 17:33:36.489 [tio-group-112] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 接收用户处于离线状态,toUserId:10
2024-10-29 17:33:36.489 [tio-group-112] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 用户处于离线状态,userId:10
2024-10-29 17:33:50.136 [tio-group-126] ERROR org.tio.core.ReadCompletionHandler - [] server:0.0.0.0:9326, client:172.16.85.134:54586, javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknownjava.nio.HeapByteBuffer[pos=30 lim=30 cap=30]
javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknown
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:131)
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:117)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:365)
	at java.base/sun.security.ssl.Alert$AlertConsumer.consume(Alert.java:293)
	at java.base/sun.security.ssl.TransportContext.dispatch(TransportContext.java:204)
	at java.base/sun.security.ssl.SSLTransport.decode(SSLTransport.java:172)
	at java.base/sun.security.ssl.SSLEngineImpl.decode(SSLEngineImpl.java:736)
	at java.base/sun.security.ssl.SSLEngineImpl.readRecord(SSLEngineImpl.java:691)
	at java.base/sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:506)
	at java.base/sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:482)
	at java.base/javax.net.ssl.SSLEngine.unwrap(SSLEngine.java:679)
	at org.tio.core.ssl.facade.Worker.doUnwrap(Worker.java:276)
	at org.tio.core.ssl.facade.Worker.unwrap(Worker.java:389)
	at org.tio.core.ssl.facade.Worker.unwrap(Worker.java:425)
	at org.tio.core.ssl.facade.SSLFacade.decrypt(SSLFacade.java:272)
	at org.tio.core.ReadCompletionHandler.completed(ReadCompletionHandler.java:281)
	at org.tio.core.ReadCompletionHandler.completed(ReadCompletionHandler.java:212)
	at java.base/sun.nio.ch.Invoker.invokeUnchecked(Invoker.java:129)
	at java.base/sun.nio.ch.Invoker$2.run(Invoker.java:221)
	at java.base/sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:113)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-10-29 17:33:59.012 [tio-group-141] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 接收用户处于离线状态,toUserId:2
2024-10-29 17:33:59.014 [tio-group-141] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 用户处于离线状态,userId:2
2024-10-29 17:33:59.161 [tio-group-141] ERROR com.dot.msg.chat.tio.util.TioUtil - [] 接收用户处于离线状态,toUserId:2
